### PR TITLE
switch to unprivileged database user on prod

### PR DESCRIPTION
**Pull request by syin**

### CODE REVIEW

**Comment by syin** [comment]
@countable does this implementation look okay?

File: README.md [hunk 1]
```diff
@@ -59,6 +59,32 @@ It is recommended that the Workspace is the `cit-web` folder, and not the `cit`
...
+ALTER DEFAULT PRIVILEGES FOR USER django GRANT SELECT ON TABLES TO django;
+
+CREATE USER migrator with PASSWORD '[REPLACE_WITH_REAL_PASSWORD]';
+GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public to migrator;
+ALTER DEFAULT PRIVILEGES FOR USER migrator GRANT SELECT, INSERT, UPDATE, DELETE ON TABLES TO migrator;
+ALTER DEFAULT PRIVILEGES FOR USER migrator GRANT USAGE, SELECT, UPDATE ON SEQUENCES TO migrator;
+```
+
+Add the following environment variables to `.env`:
+
```
Comment (countable) [inline README.md:80]: Can we provide a full production env template, prod.env?

Comment (syin) [reply README.md:80]: good idea, will do

Comment (countable) [reply README.md:80]: secrets can be left out ofc :)

**Pull request merged**